src/styles/helpers/color/validateColor.js

```diff
@@ -0,0 +1,10 @@
+export default function validateColor(color) {
+    const EMPTY_STRING = '';
+    if (!color) return EMPTY_STRING;
+    const s = new Option().style;
+    s.color = color;
+    const removeSpaces = colorStr => colorStr.replace(/\s+/g, EMPTY_STRING);
+    const isValidColor = removeSpaces(s.color) === removeSpaces(color);
+    const isValidHex = /^#([0-9A-F]{3}){1,2}$/i.test(color);
+    return isValidColor || isValidHex ? color : EMPTY_STRING;
+}
```

Comment:
uumm, too complicated 

```js
export default function validateColor(color) {
    const element = document.createElement('a');
    element.style.color = color;
    return element.style.color !=== '';
}
```

I'm can of confuse with the name of the function, it says validateColor which means return true|false .. but you are return empty string sometimes, it's a normalization then ??

Reply:
I see you are using it above like a normalizer ... uumm ... there are two levels of abstraction in your implementation, you are first normalizing, then in the styled component if empty you fallback to brand color ... think about it .. too many levels of abstraction make harder to understand to somebody else because there are color logic in two places

let's simplify by implementing the logic only in the styled component with a function `isColorValid`, if not then the fallback to brand color ... the avatar component just pass-through the color down ... it makes sense??